testing/emulationstation/APKBUILD

```diff
@@ -0,0 +1,63 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=emulationstation
+pkgver=2.7.1
+pkgrel=0
+pkgdesc="EmulationStation is a cross-platform graphical front-end for emulators with controller navigation."
+url="https://github.com/RetroPie/EmulationStation"
+arch="all"
+license="MIT"
+depends="$pkgname-theme-default"
+makedepends="boost-dev freeimage-dev freetype-dev sdl2-dev mesa-dev alsa-lib-dev vlc-dev curl-dev cmake"
+subpackages="$pkgname-theme-default:theme:noarch"
+options="!check"
+_distname="EmulationStation"
+_pugixml_git="https://github.com/zeux/pugixml.git"
+
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/RetroPie/${_distname}/archive/v${pkgver}.tar.gz
+	default-paths.patch
+	emulationstation.desktop
+	emulationstation.png"
+builddir="$srcdir"/$_distname-$pkgver
+
+prepare() {
+	cd "$builddir"
+        if [ ! -d external/pugixml.dist ]; then
+        mv -f external/pugixml external/pugixml.dist
+        git clone --depth=1 "$_pugixml_git" external/pugixml
+        fi
+	default_prepare
+}
+
+build() {
+	cd "$builddir"
+	cmake .
+	make
+}
+
+package() {
+	cd "$builddir"
+	mkdir -p "$pkgdir"/usr/bin
+	mkdir -p "$pkgdir"/usr/share/pixmaps
+	mkdir -p "$pkgdir"/usr/share/applications
+	mkdir -p "$pkgdir"/usr/share/${pkgname}/gamelists
+	mkdir -p "$pkgdir"/usr/share/${pkgname}/themes
+	install -Dm755 emulationstation "$pkgdir"/usr/bin/emulationstation
+	install -Dm644 "$srcdir"/${pkgname}.png "$pkgdir"/usr/share/pixmaps/
+	install -Dm644 "$srcdir"/${pkgname}.desktop "$pkgdir"/usr/share/applications/
+}
+
+theme() {
+	local _theme_git="https://github.com/rxbrad/es-theme-gbz35.git"
+	local _theme_name="gbz35"
```

Comment:
Should be pulled through `source=""`, would be great if you could get upstream to make a tagged release and use that instead.

Reply:
Done.